conf/swisnap-init.sh

```diff
@@ -238,6 +239,16 @@ run_plugins_with_default_configs() {
 
 }
 
+run_plugins_customizations() {
+    if [[ "${SWISNAP_CUSTOMIZE_ELASTICSEARCH}" == "true" ]] && [ -f "/tmp/customize_elasticsearch.sh" ]; then
```

Comment:
please add a comment explaining when does this file exist

Reply:
Added